rotkehlchen/chain/evm/decoding/velodrome/decoder.py

```diff
@@ -247,8 +249,3 @@ def _decode_gauge_events(self, context: DecoderContext) -> DecodingOutput:
                     event.notes = f'Receive {event.balance.amount} {crypto_asset.symbol} rewards from {gauge_address} {self.counterparty} gauge'  # noqa: E501
 
         return DecodingOutput(refresh_balances=found_event_modifying_balances)
-
-    def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
```

Comment:
So question. This is no longer needed because it happens in `ReloadableDecoderMixin::reload_data` right?

Reply:
yeah, ReloadableDecoderMixin::reload_data just returns the new_cache_data diff and then it's merged in EVMTransactionDecoder::reload_data